client/src/components/breadcrumbs.jsx

```diff
@@ -0,0 +1,46 @@
+// eslint-disable-next-line no-unused-vars
+import React, { useState, useEffect } from 'react'
+import { useLocation } from 'react-router-dom'

```

Comment:
i don't think we need react-router-dom at all for implementing breadcrumbs component.
it doesn't deal with multiple pages or any routes.